README.md

```diff
@@ -56,4 +57,34 @@ npx lerna create '@chanzuckerberg/czedi-kit-<package-name>' \
 
 ### Publishing
 
-We're not ready to publish yet
+**We are only publishing alpha pre-releases at this time**
+
+1. Confirm that all checks are green on CI.
+2. Run `git checkout master`
+3. Increment `<version>` based on the latest alpha version:
+
+```bash
+npx lerna version --no-push --conventional-commits --cd-version prerelease --message "chore(release): publish v0.0.1-alpha.<version>"
+```
+
+`--no-push` ensures the commit is not auto-pushed to remote git.
+
+3. Run
+
+```bash
+npx lerna publish from-git --no-push
+```
+
+and confirm that tags/commit/changelog etc. look correct
+
+4. Push commit and tags to remote:
+
+```bash
+git push origin --tags
+```
+
+5. Finally, run `git push origin master` (this may not be strictly necessary, but doesn't hurt)
```

Comment:
Does the `git push origin --tags` step push only tags, or also the commits that get created by lerna? If it only pushes tags, then I think this step will be necessary.

Reply:
ah good call, it looks like it only pushes tags